dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,127 @@
+| stop    | `dmctl relay stop --source-name="source1.yaml" --worker-name="worker1"`                                                                 | --source-name, --worker-name          | stop relay for a source on a worker.                                         |
+| purge   | `dmctl relay purge --source-name="source1.yaml" --file-name="mysql-bin.000006" --sub-dir="2ae76434-f79f-11e8-bde2-0242ac130008.000001"` | --source-name, --file-name, --sub-dir | purges relay log files of the DM-worker according to the specified filename. |
+
+### dmctl commands for DDL-LOCK
+
+| command | Full Syntax Example                                          | Flags                    | Description                                  |
+| ------- | ------------------------------------------------------------ | ------------------------ | -------------------------------------------- |
+| list    | `dmctl ddl-lock list --task-name="test"`                     | --task-name              | show shard-ddl locks information for a task. |
+| unlock  | `dmctl ddl-lock unlock --task-name="test" --lock-id="lock1"` | --task-name, --lock-name | force unlock un-resolved DDL locks.          |
+
+### dmctl commands for member
+
+| command                    | Full Syntax Example                                                 | Flags          | Description                                       |
+| -------------------------- | ------------------------------------------------------------------- | -------------- | ------------------------------------------------- |
+| member list                | `dmctl member list --name="master-1" --role="master/worker/leader"` | --name, --role | show members of current cluster by name and role. |
+| member offline             | `dmctl member offline --name="master-1"`                            | --name         | offline members of current cluster by name.       |
+| member evict-leader        | `dmctl member evict-leader --name="master-1"`                       | --name         | evict leader for master node.                     |
+| member cancel-evict-leader | `dmctl member cancel-evict-leader --name="master-1"`                | --name         | cancel evict leader for master node.              |
+
+### Optimised dmctl for interaction mode (optional)
+
+The current interaction of dmctl still has room for optimization, and I hope to take this opportunity to do some optimization of the interaction experience, focusing on the user can quickly select the command they want to enter through the keyboard, instead of entering commands by their memory.
+
+Here is a simple prototype demo:
+
+[![asciicast](https://asciinema.org/a/AVnOlghSu5d0YqjdOw9iruFdW.svg)](https://asciinema.org/a/AVnOlghSu5d0YqjdOw9iruFdW)
```

Comment:
this recording has been archived... I think you can push the SVG also.

Reply:
thx  added in https://github.com/pingcap/tiflow/pull/4458/commits/4aabce25879b45688ca0fd760744fb646ade9510